examples/rbac_model.conf

```diff
@@ -11,4 +11,4 @@ g = _, _
 e = some(where (p.eft == allow))
 
 [matchers]
-m = g(r.sub, p.sub) && r.obj == p.obj && r.act == p.act
\ No newline at end of file
+m = g([r.sub, p.sub]) && r.obj == p.obj && r.act == p.act
```

Comment:
Why change here? We must keep the same model grammar with all Casbin implementations

Reply:
Hi @hsluoyz,
Dart does not support variable number of arguments. So, we have to send a list wherever we need them.

Reply:
I am not sure if there is any case where we need to send in more than 2 parameters for the GFunction. If it always needs just 2 parameters, we can fix GFunction to have only 2 parameters and in that case, we won't need to send a list.

Reply:
I'm not sure if there is any case where we can send in more than 3 parameters. But if GFunction needs only 3 parameters with domain as optional. We can fix it to have 2 required parameters and 1 optional parameter. This way we can retain the original grammar.

Reply:
I'm talking about the compatibility between all Casbin implementations. All Casbin like Go, Java, Node.js and Dart must follow the same grammar, which is documented in official docs: https://casbin.org/docs/en/rbac

It can regarded as the "requirement document", and software design must follow the "requirement document". We can choose a best implementing way to fulfill the "requirement document".

![image](https://user-images.githubusercontent.com/3787410/122646978-8f813600-d154-11eb-88b8-c4d700ec59e3.png)